client/src/views/WorkspaceDetail.vue

```diff
@@ -69,7 +69,11 @@ export default {
           graphs { name }
         }
       }`});
-      this.tables = response.data.data.workspaces[0].tables.map(table => table.name);
+      this.tables = response.data.data.workspaces[0].tables.map(table => {
+        let name = table.name;
+        let displayName = name.includes("_") ? name.replace(/_/g, " ") : name;
```

Comment:
Two things:
1. Why are we changing underscores to spaces in table names?
2. I don't think you need to check for underscores before performing the substitution; you can just say

```javascript
const displayName = name.replace(/_/g, " ");
```

Reply:
re 1) - I made a display name variable that had a space instead of an underscore only for readability.

Reply:
My intuition is that this will eventually lead to trouble. I'd say keep the underscores in the name, even if it looks better with spaces.

Reply:
That totally makes sense  - I reverted to the original name containing the underscore

Reply:
Ah gotcha.